dist/components/progress-steps/_progress-steps.scss

```diff
@@ -49,24 +53,22 @@ $progress-steps__divider-border-color: hsl(360, 100%, 100%) !default;
         padding-right: 0; // 2
         padding-left: $progress-steps__divider-width + $progress-steps__item-padding; //2
     }
-}
 
 
-// Arrow Shape
-// ---
-//
-// We are using the two (before/after) pseudo elements to create a CSS triangle
-// for our progress step item arrow. We use two on top of each other to create
-// visual separation appearing as either a border or space.
-//
-// The `before` is used as the border or space
-//
-// The `after` is used to create the arrow shape of the `.c-progress-steps__item`
-//
-// 1. The "thickness" of the border/seperator is an illusion created by
-//    this margin-left property
+    // Progress Steps Items: Arrow Shape
```

Comment:
The reasoning would be explained in the comment (idealy) or by the heading.

For example "Arrow Shape" suggestions that the big block following is dedicated to creating the arrow shape, which is rather distinct from just "Progress Step Item"